Add anomaly detection faq (#234)
centreon/centreon-documentation: en/monitoring/anomaly-detection.md

Sending data from 2020-04-04T00:00:00 to 2020-04-05T00:00:00
Sending data from 2020-04-05T00:00:00 to 2020-04-06T00:00:00
```
## FAQ

### What services are offered and their associated SLA?

The anomaly detection service is currently in closed beta test phase as described in the Centreon documentation. The

> purpose of this closed beta test phase is to test our algorithms for calculating predictions (floating thresholds).

Reviewer: ```suggestion
purpose of this phase is to test our algorithms and their resulting predictions (floating thresholds).
```

> During this phase, Centreon will improve the anomaly detection functionality based on feedback. No SLA will be available

Reviewer: ```suggestion
During this phase, Centreon will improve the anomaly detection feature based on users feedback. No SLA will be available
```

during this phase.

### What are the selection criteria for the beta test program? for what duration and volume?

> A participation form will be available only for Centreon customers. From the responses, Centreon will select the

Reviewer: ```suggestion
A participation form is available to Centreon customers. Based on the answers to the form, Centreon will select the
```

candidates to participate in the closed beta test phase.

### What is the defined process for the beta test program?

Once the candidates have been selected, Centreon will contact them to indicate the procedure to follow. Regular meetings
will be scheduled to assess the effectiveness of the feature on the customers' platform. Remote handling sessions may be
necessary.
An update of the functionality may also be requested to validate the changes made following the feedback.

### How long is the beta test program?

> The closed beta test period is around 4 months. This should validate the predictions generated from the calculation of

Reviewer: ```suggestion
The closed beta test period will last for about 4 months. This should validate the predictions generated from the calculation of
```

the models, as well as the generation of alerts in the event console and the triggering of the notification process.

#### When will the feature be available? And for what Centreon Edition?

The Anomaly Detection functionality will be available for Centreon Business Edition in the 20.10 version.

### How long is the data stored?

> The data is kept for the entire validity of the license. They will allow to recalculate models if necessary. An

Reviewer: ```suggestion
The data is kept for the entire validity of the license. It will allow to recalculate models if necessary. An
```

> additional period of 3 months will be added at the end of the validity of the license before deletion of the latter.

Reviewer: ```suggestion
additional period of 3 months will be added at the end of the validity of the license before its deletion.
```

### What data is hosted by the service?

> The data hosted by the anomaly detection service only concerns data linked to the services created by the user. They

Reviewer: ```suggestion
The data hosted by the anomaly detection service only concerns data linked to the services created by the user. It
```

> include the date and time of the control, the identifier of the monitored resource, the identifier of the associated

Reviewer: ```suggestion
includes the date and time of the control, the identifier of the monitored resource, the identifier of the associated
```
